src/paid-cookie-consent/guidance-opportunity-mapper.js

```diff
@@ -132,11 +132,13 @@ export function mapToPaidOpportunity(siteId, url, audit, pageGuidance) {
 }
 
 export async function mapToPaidSuggestion(context, siteId, opportunityId, url, pageGuidance = []) {
+  const { Suggestion } = context.dataAccess;
   return {
     opportunityId,
     type: 'CONTENT_UPDATE',
     rank: 1,
-    status: 'NEW',
+    status: context.site?.requiresValidation ? Suggestion.STATUSES.NOT_VALIDATED
```

Comment:
To me, it looks that the cookie-consent is always marking the old opportunity entries as Ignored and creating new entries.

After your PR changes, would the paid-cookie-consent opportunity, even if existing, always be re-created with Not_Validated suggestion?

That looks like the wrong experience.